### DISCUSSION TITLE
How to run my cpp file on code-server

### DISCUSSION THREAD

**Discussion by cevinov** [coder/code-server#4987] [Q&A]
Hello sir,

I wanna ask how can i run my cpp file ?.

Footage :
https://screenrec.com/share/MvlaEdqtnb

Thanks

**Comment by lg-nineoneninethreenine** [comment]
You need to run it with the `g++` command on Terminal if I am not wrong. To get debug tools, try installing the extension from the VSIX. Can't tell how right now, will tell soon.

**Reply by cevinov** [reply]
okay sir, wait for your replies
